### ISSUE TITLE
Geth error : replacement transaction too low fee

### ISSUE THREAD

**Issue by kyriediculous** [javahippie/geth-dev#48]
Or something along those lines.

This happens when executing certain transactions when migrating or testing when the gasPrice is set to 0 or the client doesn't reprice transactions that failed to execute when sent again. 

Therefore I propose getting rid of 'stuck' transactions in the nodes since this is a network for development purposes and resetting the chain everytime it occurs might not be optimal. 

`--txpool.lifetime '0h0m5s' `(default is 3h)

**Comment by javahippie** [comment]
This sounds really helpful, but I am not too sure about the 5 seconds. This is exactly the configured block time of our network, maybe we should configure `--txpool.lifetime '0h0m10s'` ?

**Comment by kyriediculous** [comment]
If I'm not mistaken geth makes the difference between pending and unexecuted meaning the 5 seconds only starts to run after the transaction has not been included in the next block by the chosen sealer. 

I think it should even work when putting it to 0.

**Comment by javahippie** [comment]
Yes, I misunderstood the timer there, the documentation also mentions "non-executable transactions" 

https://github.com/ethereum/go-ethereum/wiki/Command-Line-Options
